R/dfm.R

```diff
@@ -313,11 +324,11 @@ dfm.dfm <- function(x,
     
     if (!is.null(groups)) {
```

Comment:
if `tokens_group` + dfm is faster than dfm + `dfm_group`, then we should do that sequence internally in `dfm.tokenizedTexts()`, per issue #498.